src/components/todoList/TodoListForm.tsx

```diff
@@ -19,25 +19,25 @@ export function TodoForm() {
     },
   );
 
-  const handleChange = (e) => {
+  const handleChange = (e:any) => {
     const { value } = e.target;
-    const id = Date.now();
 
     setFormInputData({
       ...formInputData,
-      id,
+      id:Date.now().toString(),
```

Comment:
Why is the id value the current date? Why don't you use a counter or something instead? Maybe create a DexieJS collection and use that instead of manually defining everything. And please add some space between the colon and the text

Reply:
The id is calculated in milliseconds and they will never be the same when the user adds goals...I think its the most efficient id generator.

Reply:
Since we want to be able to distinguish events across many different users so that they can coordinate automatically I would suggest a UUIDv4 to make sure there is no conflict. A ms conflict is rare - but if you have millions of users - it will happen.

Agreed, the ms is very efficient - and fine for a single user - but in this case avoiding conflicts across users is more important than efficiency.

Reply:
I've changed it now...I think the uuid was the best after all.